<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>35</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Choose OS(Exclusive)</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="windowsCheckBox">
               <property name="text">
                <string>Windows</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="macCheckBox">
               <property name="text">
                <string>Mac</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="linuxCheckBox">
               <property name="text">
                <string>Linux</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Choose Drink (Non Exclusive)</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="beerCheckBox">
               <property name="text">
                <string>Beer</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="juiceCheckBox">
               <property name="text">
                <string>Juice</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="coffeeCheckBox">
               <property name="text">
                <string>Coffee</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Choose Answer</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="aRadioButton">
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="bRadioButton">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="cRadioButton">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="grabDataButton">
        <property name="text">
         <string>Grab Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setStateButton">
        <property name="text">
         <string> Set State</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
